src/main/java/com/eva/MainApp.java

```diff
@@ -82,25 +83,32 @@ private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
         ReadOnlyEvaDatabase<Person> initialPersonData;
```

Comment:
Not sure if it's ok to delete the person database. If we can do so without the application collapsing we should remove it if it is redundant! (of course this means updating all the tests) KIV